package.json

```diff
@@ -293,20 +294,25 @@
                     "when": "view == dockerContainers && viewItem =~ /container$/i",
                     "group": "containers_1_general@3"
                 },
+                {
+                    "command": "vscode-docker.containers.browse",
```

Comment:
Should we show this only in context menu or also in palette? I'm OK with showing in palette, just something to think about.

Reply:
Its peer commands seem to be designed to support the palette, so I'm ok with that, too.

Reply:
Makes sense. It might also be the case that they're in the debugging window and don't want to switch.